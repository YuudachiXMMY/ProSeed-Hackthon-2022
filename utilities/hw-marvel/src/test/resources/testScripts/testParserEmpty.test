# Test Marvel parser can load an empty graph

LoadGraph emptyGraph empty.tsv

ListNodes emptyGraph