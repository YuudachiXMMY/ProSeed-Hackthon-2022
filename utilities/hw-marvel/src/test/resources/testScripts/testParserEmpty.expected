# Test Marvel parser can load an empty graph

loaded graph emptyGraph

emptyGraph contains: